Formula/c/cortexso.rb

```diff
@@ -0,0 +1,34 @@
+require "language/node"
+
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://github.com/janhq/cortex.git",
+      tag:      "v0.4.20",
```

Comment:
since 20 did not get marked as latest release, maybe use 15?
```suggestion
      tag:      "v0.4.15",
```

Reply:
Thanks for reminding, we have updated to use v0.4.16, which is marked as the latest release